openlabcmd/openlabcmd/plugins/nodepool/network.py

```diff
@@ -32,7 +31,7 @@ def check(self):
             return
 
         # get subnet of openlab-net successfully, check subnet
-        if "openlab-subnet 192.168.199.0/24" not in res:
+        if "openlab-subnet 192.168.0.0/24" not in res:
```

Comment:
all ip should be in `192.168.0.0`? I see you also used `10.3.0.0` and `192.168.199.0`

Reply:
10.3.0.0 is old net cidr in citynetwork and 192.168.199.0 is old cidr in huaweicloud. After reform, we change these all providers to 192.168.0.0/24 cidr

Reply:
Got it

Reply:
Have you attached the new port on the citynetwork ha nodes?